bindings/cs/MongoDB.Crypt.Example/Program.cs

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- 
+
```

Comment:
There are two changes types in the below files
1. Replacing a single income parameter in `CryptOptions` on an array with one item (for example line `104`).
2. All other changes are autoformatting.